### PR TITLE
ossgg: side quest social-poster-here link add to file

### PR DESCRIPTION
**Pull request by adityadeshlahre**
- #6912 

### Social hero
image

### Social Poster
image

### Financial Blog 
- [article](https://dev.to/adityadeshlahre/redefining-financial-analysis-with-openbb-the-power-of-open-source-versus-proprietary-tools-2j2p)

### Fundamental Blog
https://dev.to/adityadeshlahre/streamlining-research-and-analysis-with-openbb-209h

### CODE REVIEW

**Review by jmaslek — APPROVED** [review]

**Comment by adityadeshlahre** [comment]
hey @jmaslek i think you have missed my point for article :slightly_smiling_face:

**Comment by piiq** [comment]
The blog post is deeply wrong factually. It is not accepted. Please remove it from the submission if you want this PR to be merged by the end of the day

**Comment by piiq** [comment]
nvm, I did it for you

File: oss.gg/no_code_side_quests/5-financial-post.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Your turn 👇
 » 28-October-2024 by [Atharva Deshmukh](https://oss.gg/Atharva-3000)
 » Link to Article [HERE](https://dev.to/atharva3000/openbb-vs-proprietary-tools-why-open-source-is-the-future-of-financial-analysis-391a)
 
+» 30-October-2024 by aditya deshlahre » Link to Article: [article](https://dev.to/adityadeshlahre/redefining-financial-analysis-with-openbb-the-power-of-open-source-versus-proprietary-tools-2j2p)
```
Comment (piiq) [inline oss.gg/no_code_side_quests/5-financial-post.md:27]: Remove this post from the submission please. It is not accepted

**Review by piiq — CHANGES_REQUESTED** [review]
Remove the dev.to post from the PR please

**Review by piiq — APPROVED** [review]

**Pull request merged**